### ISSUE TITLE
S1185 false negative for member with summary

### ISSUE THREAD

**Issue by ds1709** [SonarSource/sonar-dotnet#5318]
### Description

Rule S1185 not fires if overriden member has summary.

### Repro steps

```c#
    public class Foo
    {
        public virtual int Bar(int a, int b)
        {
            return a + b;
        }
    }

    public class Foo2 : Foo
    {
        public override int Bar(int a, int b)  // warning
        {
            return base.Bar(a, b);
        }
    }

    public class Foo3 : Foo
    {
        /// <summary>
        /// 
        /// </summary>
        /// <param name="a"></param>
        /// <param name="b"></param>
        /// <returns></returns>
        public override int Bar(int a, int b) // no warning
        {
            return base.Bar(a, b);
        }
    }
```

### Expected behavior

Warning fires no matter is there summary or not.

### Actual behavior

Warning fires only if member has no summary.

### Related information

* Visual Studio Enterprice 2022
* SonarAnalyzer.CSharp 8.34.0.42011
* Windows 10 Corporate 21H1

**Comment by pavel-mikula-sonarsource** [comment]
Hi @ds1709,

This behavior is by-design and described in `Exceptions` section of the [S1185 rule description](https://rules.sonarsource.com/csharp/RSPEC-1185).
